### PR TITLE
Add event-target-shim to LogProxy

### PR DESCRIPTION
**Pull request by mofojed**
Needed for Safari <14

### CODE REVIEW

**Comment by mofojed** [comment]
I ran `./node_modules/.bin/lerna add event-target-shim --scope=@deephaven/log ` to add the event-target-shim lib, so I think the updates to the package-lock.jsons is also correct (removing the common packages from each package-lock.json to have it only in the highest level package-lock.json)

File: packages/code-studio/package-lock.json [hunk 1]
```diff
@@ -1282,124 +1264,6 @@
...
-		"@deephaven/chart": {
-			"version": "0.1.1",
-			"resolved": "https://registry.npmjs.org/@deephaven/chart/-/chart-0.1.1.tgz",
-			"integrity": "sha512-QH7Zgp4K5+NeRZCTdjTXK3gen91NfUB/Fw0oCuvcCuvWciegJXHW/RIeW9isCPUt6F7iB7K/fMTLv3Zxyz0odA==",
-			"requires": {
-				"plotly.js": "^1.54.7",
-				"react-plotly.js": "^2.4.0"
-			}
-		},
-		"@deephaven/components": {
```
Comment (mattrunyon) [inline packages/code-studio/package-lock.json:1294]: Weird that all our packages were removed from this package-lock

**Review by mattrunyon — APPROVED** [review]
Make sure our packages should have been removed from code-studio package lock

File: packages/code-studio/package-lock.json (hunk 1 above)
Comment (mofojed) [reply packages/code-studio/package-lock.json:1294]: I believe it is correct - since it's pulling from within the same repo

**Comment by mofojed** [comment]
Fixes #51

**Pull request merged**